tests/ToolInfoTest.php

```diff
@@ -21,13 +21,87 @@
  */
 final class ToolInfoTest extends \PHPUnit_Framework_TestCase
 {
+    const PHPUNIT_BIN = '/../vendor/bin/phpunit';
+
+    protected function setUp()
+    {
+        foreach ($this->provideScriptName() as $testArguments) {
+            $symlink = current($testArguments);
+            if ($symlink === self::PHPUNIT_BIN or is_link(__DIR__.$symlink)) {
```

Comment:
Never intented to troll or change the status quo.
For sure consistency and homogeneity come first: will change in next commits.
My OT was more generic, outside this project.